src/site/xdoc/checks/regexp/regexpsingleline.xml.template

```diff
@@ -92,20 +101,20 @@
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsingleline/Example6.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsingleline/Example6.java"/>
           <param name="type" value="config"/>
         </macro>
-        <p id="Example6-code">Example:</p>
+        <p id="Example6-raw">Example:</p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsingleline/Example6.txt"/>
-          <param name="type" value="code"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsingleline/Example6.sql"/>
+          <param name="type" value="raw"/>
         </macro>
-        <p id="Example7-code">Example:</p>
+        <p id="Example7-raw">Example:</p>
```

Comment:
`Example with violation:`

Reply:
Please change wording for all examples that has two examples in row , one for no violations, second is for violation

Reply:
> Please change wording for all examples that has two examples in row , one for no violations, second is for violation

Done

Reply:
Not done. Please make sure you pushed.

All examples with violations should be "example with violations:` wording

Reply:
> Not done. Please make sure you pushed.
> 
> All examples with violations should be "example with violations:` wording

Ok. Should be done now. I updated the wordings in `regexsingleline.xml.template`, but it also needed to be updated in `regexsingleline.xml` (which I think wasn't being updated due to the failing test).